.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
I can remove this, the intent was so that dist can still be committed just not by default. Not sure it actually helped anyway since dist/ is now used as the demo assets dir.